src/classification.jl

```diff
@@ -57,6 +73,20 @@ function filter_walkways(ways::Vector{OpenStreetMapX.Way},classes::Dict{String,
     return walkways
 end 
 
+"""
+    classify_walkways(ways::Vector{OpenStreetMapX.Way},
+                      classes::Dict{String, Int} = OpenStreetMapX.PED_CLASSES)
+					
+Classifies a vector of OpenStreetMapX ways based on their pedestrian attributes.
+It considers the presence of a "sidewalk" tagand checks if the corresponding value or the "highway" tag value
+is present in the specified classes dictionary
+
+**Arguments**
+
+* `ways::Vector{OpenStreetMapX.Way}` : vector of Way struct
```

Comment:
`Vector` of `Way`s
1. Use apostrophe formatting whenever referencing data structures
2. All Julia datatypes except for primitives are `struct`s so no need to write that (remove the word struct(